packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -67,8 +76,7 @@ export class ProxyController extends EventEmitter {
 			workers: [
 				{
 					name: "ProxyWorker",
-					// `url_standard` required to parse IPv6 hostnames correctly
-					compatibilityFlags: ["nodejs_compat", "url_standard"],
+					compatibilityFlags,
```

Comment:
Why does this worker have an old compatibility date? Would it be possible to update it to a more recent compatibility date and avoid the need for some of these flags?